2d_segmentation/torch/unet_evaluation_array.py

```diff
@@ -61,22 +61,21 @@ def main(tempdir):
     model.load_state_dict(torch.load("best_metric_model_segmentation2d_array.pth"))
     model.eval()
     with torch.no_grad():
-        metric_sum = 0.0
-        metric_count = 0
         saver = PNGSaver(output_dir="./output")
         for val_data in val_loader:
             val_images, val_labels = val_data[0].to(device), val_data[1].to(device)
             # define sliding window size and batch size for windows inference
             roi_size = (96, 96)
             sw_batch_size = 4
             val_outputs = sliding_window_inference(val_images, roi_size, sw_batch_size, model)
-            val_outputs = post_trans(val_outputs)
-            value, _ = dice_metric(y_pred=val_outputs, y=val_labels)
-            metric_count += len(value)
-            metric_sum += value.item() * len(value)
+            val_outputs = torch.stack([post_trans(i) for i in decollate_batch(val_outputs)])
```

Comment:
No need to `torch.stack()`

Reply:
Here `PNGSaver` is used, and it needs tensor/numpy array input

Reply:
`SaveImage` transform also support PNG format, could you please change to use `SaveImage` instead?